src/Nethermind/Nethermind.Merge.Plugin/Handlers/ForkchoiceUpdatedHandler.cs

```diff
@@ -284,7 +284,7 @@ protected virtual bool IsOnMainChainBehindHead(Block newHeadBlock, ForkchoiceSta
         if (shouldUpdateHead)
         {
             _poSSwitcher.ForkchoiceUpdated(newHeadBlock.Header, finalizedBlockHash);
-            if (_logger.IsInfo) _logger.Info($"Synced Chain Head to {newHeadBlock.ToString(Block.Format.Short)}");
+            if (_logger.IsInfo) _logger.Info($"Updated Chain Head to {newHeadBlock.ToString(Block.Format.Short)}");
```

Comment:
Adds an extra char to the logs; so now the block number for Updated will be out of alignment with all the other logs

<img width="307" alt="image" src="https://github.com/user-attachments/assets/22932de9-88f8-4050-8c53-f02b0c15b209" />

Either need to add an additional space to all those logs or use a word with one less character

Reply:
Everything aligned (red line)

![image](https://github.com/user-attachments/assets/333d376f-070a-4395-931d-f85acec81682)